Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -183,43 +193,52 @@ function (array $matches) use (&$unresolvedUris, $absolute, $nodeAddress) {
     /**
      * Replace the target attribute of link tags in processedContent with the target
      * specified by externalLinkTarget and resourceLinkTarget options.
-     * Additionally set rel="noopener" for links with target="_blank".
+     * Additionally sets rel="noopener" for links with target="_blank",
+     * and rel="noopener external" for external links.
      */
     protected function replaceLinkTargets(string $processedContent): string
     {
-        $noOpenerString = $this->fusionValue('setNoOpener') ? ' rel="noopener"' : '';
+        $setNoOpener = $this->fusionValue('setNoOpener');
+        $setExternal = $this->fusionValue('setExternal');
         $externalLinkTarget = trim($this->fusionValue('externalLinkTarget'));
         $resourceLinkTarget = trim($this->fusionValue('resourceLinkTarget'));
         if ($externalLinkTarget === '' && $resourceLinkTarget === '') {
             return $processedContent;
         }
         $controllerContext = $this->runtime->getControllerContext();
         $host = $controllerContext->getRequest()->getHttpRequest()->getUri()->getHost();
+        // todo optimize to only use one preg_replace_callback for uri converting
         $processedContent = preg_replace_callback(
             '~<a.*?href="(.*?)".*?>~i',
-            function ($matches) use ($externalLinkTarget, $resourceLinkTarget, $host, $noOpenerString) {
+            function ($matches) use ($externalLinkTarget, $resourceLinkTarget, $host, $setNoOpener, $setExternal) {
                 list($linkText, $linkHref) = $matches;
                 $uriHost = parse_url($linkHref, PHP_URL_HOST);
                 $target = null;
-                if ($externalLinkTarget !== '' && is_string($uriHost) && $uriHost !== $host) {
+                $isExternalLink = is_string($uriHost) && $uriHost !== $host;
+                if ($externalLinkTarget !== '' && $isExternalLink) {
                     $target = $externalLinkTarget;
                 }
-                if ($resourceLinkTarget !== '' && strpos($linkHref, '_Resources') !== false) {
+                if ($resourceLinkTarget !== '' && str_contains($linkHref, '_Resources')) {
                     $target = $resourceLinkTarget;
                 }
                 if ($target === null) {
                     return $linkText;
                 }
-                if (preg_match_all('~target="(.*?)~i', $linkText, $targetMatches)) {
+                // todo merge with "rel" attribute if already existent
+                $relValue = $isExternalLink && $setNoOpener ? 'noopener ' : '';
+                $relValue .= $isExternalLink && $setExternal ? 'external' : '';
+                $relValue = ltrim($relValue);
+                if (str_contains($linkText, 'target="')) {
+                    // todo shouldn't we merge the current target value
```

Comment:
upsi yes that seems to be a thing. And i have the fear that this problem just got worse not better ^^ https://github.com/neos/neos-development-collection/issues/2962

Reply:
mh.. that's what happens if you turn a simple "URI converter" to an "URI converter and link attribute converter and link attribute adder" swiss army knife..
I would suggest to extract the latter at some point